### PR TITLE
Fix a small description typo in constraints metrics.

### PR DESCRIPTION
**Pull request by MichaelGrupp**
My fault. 2D version is fine.

### CODE REVIEW

File: cartographer/common/port.h [hunk 1]
```diff
@@ -17,13 +17,12 @@
...
 #define CARTOGRAPHER_COMMON_PORT_H_
 
-#include <cinttypes>
-#include <cmath>
-#include <string>
-
 #include <boost/iostreams/device/back_inserter.hpp>
 #include <boost/iostreams/filter/gzip.hpp>
 #include <boost/iostreams/filtering_stream.hpp>
+#include <cinttypes>
```
Comment (gaschler) [inline cartographer/common/port.h]: style: system header before boost

File: cartographer/mapping/internal/3d/pose_graph_3d.cc [hunk 2]
```diff
@@ -1254,7 +1254,7 @@ void PoseGraph3D::RegisterMetrics(metrics::FamilyFactory* family_factory) {
   kWorkQueueDelayMetric = latency->Add({});
   auto* constraints = family_factory->NewGaugeFamily(
       "mapping_3d_pose_graph_constraints",
-      "Current number constraints in the pose graph");
+      "Current number of constraints in the pose graph");
```
Comment (gaschler) [inline cartographer/mapping/internal/3d/pose_graph_3d.cc]: Have this in this change, and the syntax style cleanup in a separate change?

**Comment by MichaelGrupp** [comment]
@gaschler I only added the "of". Anything else was done in the automatic (?) "Ran Clang-Format." commit.

**Comment by MichaelGrupp** [comment]
This also happened to other open PRs here and in Cartographer ROS.

**Comment by MichaelGrupp** [comment]
Could it be that your bot uses a different version of clang-format now?

**Comment by gaschler** [comment]
@MichaelGrupp Seems like the formatting bot went crazy here. It is restarted, could you please force push your change? Thanks

**Comment by MichaelGrupp** [comment]
Done.

Cool to see that Travis is smart enough to not re-run the build if the commit was already built.

**Pull request merged**